src/cli/commands/block/rm.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const Command = require('ronin').Command
+const utils = require('../../utils')
+const bs58 = require('bs58')
+const debug = require('debug')
+const log = debug('cli:block')
+log.error = debug('cli:block:error')
+
+module.exports = Command.extend({
+  desc: 'Remove a raw IPFS block',
+
+  options: {},
+
+  run: (key) => {
+    if (!key) {
+      throw new Error("Argument 'key' is required")
+    }
+
+    utils.getIPFS((err, ipfs) => {
+      if (err) {
+        throw err
+      }
+
+      if (utils.isDaemonOn()) {
+        // TODO implement this once `js-ipfs-api` supports it
+        throw new Error('rm block with daemon running is not yet implemented')
```

Comment:
There's no ipfs block del or rm on `go-ipfs`, so `js-ipfs-api` doesn't have it either. I could make the raw request here, but thought it would be better to wait and avoid Yet Another Hack™. What are your thoughts on this?